Automattic/wp-calypso
# Rename plans to Personal / Premium / Business / Commerce

## Proposed Changes

* As per the decision in p58i-hH2-p2, we will revert the plan names to Personal / Premium / Business / Commerce.
* This PR also removes all the experiment related code.

## Why are these changes being made?


* Due to the decision made in p58i-hH2-p2

## Testing Instructions



* First, apply D156184-code and sandbox public-api.wordpress.com
* Verify that in any screen that shows plan names, the new names are visible. Check this in signup flow plan step, `/plans` page, upsells in the Tools section, /themes page, checkout page after adding a plan to cart, and anywhere else you can think of.
* Purchase any plan and verify that the billing emails you receive all mention the new plan name Personal/Premium etc.
